Agents/PropertyValueEstimationAgent/setup.py

```diff
@@ -0,0 +1,34 @@
+from setuptools import setup, find_namespace_packages
+
+setup(
+    name='propertyvalueestimation',
```

Comment:
I noticed the folder name for your source code is `agent`, I would recommend using a more specific name that aligns with the one you use in the `setup.py`. Otherwise, it will get mixed up if you have more than one agent installed in the same environment, which will be relevant when you want to have an integration test for all derivation agents in the chain, and you will need to import all of them from agent...

Reply:
The idea behind the name discrepancy between the repository and setup.py was the following: Having the general name `agent` for the repo allows to have general launch.json, tasks.json, etc. required for the stack. The specific name in the setup.py should still allow to import several agents with their respective functionality by name. I just realise that I might have missed the relevant import statement in the `__init__.py` in the top level `agent` repo:
```
from . import agentlogic
from . import datamodel
```
Shouldn't that work without issues even when having multiple agents in the same environment? Or do you see issues with that?

Re the integration test for chained derivations: Wouldn't it be possible to deploy each agent as separate Docker container and "simply" track the expected changes within the KG to verify the correct execution of the chained derivations?

Reply:
I think if you name it differently (`propertyvalueestimation` and `agent`), then it will run in to exception below?
```sh
>>> from propertyvalueestimation import *
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'propertyvalueestimation'
```